_posts/2020-04-29-newpipe-0.19.3-release.md

```diff
@@ -0,0 +1,61 @@
+ 
+The title of our app, NewPipe, is shown in the upper left corner when you open the sidebar. Sadly on some devices with custom fonts it was forced to be in two lines, not anymore, thanks to the fourth contribution of [@mauriciocolli](https://github.com/mauriciocolli). The resizing of the pop-up window was greatly improved by [@harshlele](https://github.com/harshlele). This was a long-lasting issue and everybody should be happier now. [@Stypox](https://github.com/Stypox) found time for a second, little contribution: You can now long-press on the background and pop-up buttons in headers of channels and the videos from that channel are then added to the respective queue. [@GradyClark](https://github.com/GradyClark) made a big contribution which allows you to delete watched videos from your bookmarked playlists. Now you can easily create a watch-later playlist and occasionally delete the ones you watched from there with one button press. The contribution of [@Stypox](https://github.com/Stypox) needs to be mentioned here as well, he greatly improved the efficiency of this action.
+
+## Development
+
+This section is for internal changes with this release. To start it off, [@wb9688](https://github.com/wb9688) made a big change in enforcing Checkstyle, which should provide us with even more beautiful code then we already had in this project. Related to this, Checkstyle is also tested in the CI and UTF-8 is enforced for Gradle. [@Stypox](https://github.com/Stypox) fixed an issue where the build name was not probably overridden.
+
+The NewPipeExtractor got some improvements not mentioned so far: [@Royosef](https://github.com/Royosef) added support for PeerTube channels, [@B0pol](https://github.com/B0pol) added the supported countries for soundcloud and [@wb9688](https://github.com/wb9688) changed the `media.ccc.de` url to the new one.
+
+## Conclusion + internal news
+
+That is it for this release. We would like to use this section and your attention to provide you with some internal news: **We are looking for you!** In case you are a developer who wants to integrate the library into NewPipe. Because we need one. Please contact us via known channels.
+
+We also want to introduce the following members who have recently joined our team: [@B0pol](https://github.com/B0pol) and  [@wb9688](https://github.com/wb9688) are persistent contributors and now have an official badge. Also [@Poolitzer](https://github.com/Poolitzer) (Me. I have been here the whole time), who wrote a lot of the recent blog posts and is in general a great human being.
```

Comment:
Again a double space in there. Also mention how I was an official member a few years ago, and already added e.g. SoundCloud support back then.